packages/@lwc/engine-server/src/__tests__/fixtures/attribute-boolean/expected.html

```diff
@@ -1,5 +1,5 @@
 <x-attribute-boolean>
   <template shadowroot="open">
-    <input type="checkbox" required readonly checked>
+    <input type="checkbox" required readonly checked />
```

Comment:
I'm not super happy about this (`input` is self-closing), but Prettier doesn't seem to have an option to turn this off.